examples/experimental/dagster-airlift/dagster_airlift/core/airflow_cacheable_assets_def.py

```diff
@@ -291,8 +277,8 @@ def construct_cacheable_assets_and_infer_dependencies(
     for asset in definitions.assets:
```

Comment:
I still think we should go all the way and access the canonical definitions at `definitions.get_asset_graph().assets_defs`

Reply:
I think we need to wait for the downstream PR which doesn't expect all asset specs target the same dags/tasks first. The downstream handles that, after which I'll just do canonical definitions access.